auth_brute_force/data/ir_config_parameter.xml

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright 2015 GRAP -Sylvain LE GAL
-     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
-<odoo noupdate="1">
-
-    <data>
-
-        <record id="max_attempt_qty" model="ir.config_parameter">
-            <field name="key">auth_brute_force.max_attempt_qty</field>
```

Comment:
How do we control this now? No change in README?

Reply:
I updated the README.